src/main/java/mat/model/cql/CQLIncludeLibrary.java

```diff
@@ -25,7 +28,7 @@
 
     private String measureId;
 
-    private String libraryModelType;
+    private String libraryModelType = ModelTypeHelper.QDM;

```

Comment:
The library's model type is not persisted in Cator'x model XML. 
We should probably change that.